templates/deployment.yaml

```diff
@@ -17,6 +17,7 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
       annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
Nice, I don't know why I did not think about this earlier :blush:

Reply:
I've been trying better solutions, but I didn't found a clear way to get the hash of the configmap when it's created in the parent chart.